RELEASES.md

```diff
@@ -1,3 +1,16 @@
+## 30.0.1
+
+Released 2025-02-21.
+
+### Fixed
+
+* Fixes an issue building the `cranelift-assembler-x64` crate on Windows
+  when the Rust toolchain is on a different drive than the project using
+  `wasmtime`. For more details, see the [Zulip discussion]. [#10270]
+
+[Zulip discussion]: https://bytecodealliance.zulipchat.com/#narrow/channel/217126-wasmtime/topic/Wasmtime.2030.20x64.20assembler.20build.20error.20on.20Windows
+[#10270]: https://github.com/bytecodealliance/wasmtime/pull/10270
+
```

Comment:
Oh one final thing here, can you add a bunch of `------------------` between this and the 30.0.0 section below? You can copy/paste the line [down below](https://github.com/bytecodealliance/wasmtime/blob/fd08086eff2e12c5a647561434e9930e3d140290/RELEASES.md?plain=1#L173) up here but the purpose is to break apart sections so [we an automatically upload release notes](https://github.com/bytecodealliance/wasmtime/blob/789e7d711417059066630b5c0dd776c0a2cece69/.github/actions/github-release/main.js#L104) to each github release

Reply:
Sure